providers/src/airflow/providers/fab/www/app.py

```diff
@@ -0,0 +1,87 @@
+#
```

Comment:
I copied this file from `airflow/www/app.py` and removed what seemed to me unnecessary